assets/src/components/d3/createResourceAccessChart.js

```diff
@@ -413,7 +427,7 @@ function createResourceAccessChart ({ data, weekRange, gradeSelection, resourceT
     .text('Percentage of All Students in the Selected Grade Range')
     .style('font-size', '14px')
 
-  mainGroup.append('g')
+  mainGroup.insert('g', ':first-child')
```

Comment:
This moves the link labels before the bars in the DOM, which makes the Tab order go from left to right (expected for our users?).